tf-modules/archive/reconciliation_report.tf

```diff
@@ -20,6 +20,8 @@ resource "aws_lambda_function" "create_reconciliation_report" {
       system_bucket         = var.system_bucket
       cmr_client_id         = var.cmr_client_id
       cmr_provider          = var.cmr_provider
+      cmr_username          = var.cmr_username
+      cmr_password          = var.cmr_password
```

Comment:
same comment https://github.com/nasa/cumulus/pull/1620/files#r423855120

Reply:
This is now updated.